package.json

```diff
@@ -44,7 +44,7 @@
     "jest": "^26.6.3",
     "less": "^3.12.2",
     "node-sass": "^5.0.0",
-    "postcss": "^8.1.14",
+    "postcss": "^8.2.7",
```

Comment:
Please update to enforce at least version `8.2.10` to avoid vulnerability as per https://www.npmjs.com/advisories/1693

Reply:
It should not make much difference as the `^` in front of the version number should let you update to the latest postcss version anyway. In any case it would be better to do it in a separate PR since this one also updates cssnano which is potentially more breaking than bumping postcss version.